### PR TITLE
configurations to build on blake and white, Skylake and P100 testbeds…

### PR DESCRIPTION
**Pull request by mfdeakin-sandia**
… respectively.

Currently this fails to run on blake, and I can't test on white since the P100 nodes are down.
This also has a fix for building with separate netcdf-c and netcdf-fortran libraries

blake failure:
```
 Failed to open file idmap
 ERROR: (shr_mct_sMatReaddnc)No such file or directory
Image              PC                Routine            Line        Source             
e3sm.exe           000000000092A926  Unknown               Unknown  Unknown
e3sm.exe           0000000000556522  shr_abort_mod_mp_         114  shr_abort_mod.F90
e3sm.exe           000000000063B940  shr_mct_mod_mp_sh         513  shr_mct_mod.F90
e3sm.exe           0000000000640B00  shr_mct_mod_mp_sh         349  shr_mct_mod.F90
e3sm.exe           00000000004E6AA1  seq_map_mod_mp_se         143  seq_map_mod.F90
e3sm.exe           0000000000472DB3  prep_ocn_mod_mp_p         308  prep_ocn_mod.F90
e3sm.exe           0000000000429E03  cime_comp_mod_mp_        1626  cime_comp_mod.F90
e3sm.exe           0000000000431207  MAIN__                     92  cime_driver.F90
e3sm.exe           00000000004149DE  Unknown               Unknown  Unknown
libc-2.17.so       00002B53A4B0CC05  __libc_start_main     Unknown  Unknown
e3sm.exe           00000000004148E9  Unknown               Unknown  Unknown
```

### CODE REVIEW

File: cime/scripts/Tools/Makefile [hunk 1]
```diff
@@ -180,7 +180,7 @@ ifdef NETCDF_C_PATH
     LIB_NETCDF_C:=$(NETCDF_C_PATH)/lib
   endif
   ifndef LIB_NETCDF_FORTRAN
-    LIB_NETCDF_FORTRAN:=$(NETCDF_C_PATH)/lib
+    LIB_NETCDF_FORTRAN:=$(NETCDF_FORTRAN_PATH)/lib
```
[the remark this answers is not in the document]
Comment (sarats) [reply cime/scripts/Tools/Makefile]: Never mind, I see the error message above when NETCDF_C is defined without NETCDF_FORTRAN.

[the remark this answers is not in the document]
Comment (mfdeakin-sandia) [reply cime/scripts/Tools/Makefile]: I'm curious how e3sm builds on summit when this is wrong?

[the remark this answers is not in the document]
Comment (sarats) [reply cime/scripts/Tools/Makefile]: In the grand scheme of things, I don't know where LIB_NETCDF_FORTRAN falls along with module paths etc. But when I last checked, it picks up all the right libraries. 

I also wondered about the different ways some of these paths/libs are set in config_machines.xml for different machines.

**Comment by mfdeakin-sandia** [comment]
Merged to next. Because of the makefile change, I'm waiting until tomorrow to merge to master

**Pull request merged**